doc/engineering-reference/src/simulation-models-encyclopedic-reference-004/setpoint-managers.tex

```diff
@@ -56,7 +56,7 @@ \subsection{Outdoor Air Reset}\label{outdoor-air-reset}
 {T_{set,atOAhigh}} = {T_{2,set,atOAhigh}}
 \end{equation}
 
-otherwise
+Otherwise
```

Comment:
No changes are needed in the Engineering Reference section for SetpointManager:OutdoorAirReset.  However, the eng ref section for "Summary of Plant Loop Demand Calculation Schemes" needs a small change.

". . . For the DualSetPointDeadband scheme the Plant Loop requires one or two Setpoint Managers that set the high and low setpoint values for
Node.TempSetPointHi and Node.TempSetPointLo. SetpointManager:Scheduled:DualSetpoint sets both the high and low setpoints.  Otherwise, two setpoint managers are required, one with Control Variable = MaximumTemperature and another with Control Variable = MinimumTemperature. Examples of applicable setpoint managers include: SetpointManager:Scheduled, SetpointManager:OutdoorAirReset, SetpointManager:FollowOutdoorAirTemperature, etc.